### PR TITLE
Fixing issue #95

### PR DESCRIPTION
**Pull request by jazeee**
This is a simple fix for #95 
I noticed it during some testing.
Thanks much!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+#### Changed
```
Comment (kahmali) [inline CHANGELOG.md]: This belongs under an `## Unreleased` section (needs to be added above)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## Unreleased
+
+#### Changed
```
Comment (jazeee) [inline CHANGELOG.md]: Added ## Unreleased

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+- Minor update to ensure that for 401 and 403 errors, Restivus sends the response content after a delay.
+#### Fixed
```
Comment (kahmali) [inline CHANGELOG.md]: You can remove the "Changed" section and keep what you have in the "Fixed" section.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## Unreleased
+
+#### Fixed
+- Issue #95
+  - On 401, 403 errors, Restivus should wait before sending any data
```
Comment (jazeee) [inline CHANGELOG.md]: Updated

**Comment by kahmali** [comment]
The code looks good and works as expected. I left one other minor comment in the change log, and then the commits just need to be squashed together and rebased onto dev (it looks like you merged the latest from dev, instead of rebasing). Once that's set I'll pull this is and bump the version.

**Comment by jazeee** [comment]
I am unsure how to get rid of the merge log. Rebase seems to skip those commits in interactive mode.
I created PR #97 which replaces this one.

**Comment by kahmali** [comment]
See #97

**Pull request closed without merging**